common/contract.ts

```diff
@@ -62,6 +62,7 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   featuredOnHomeRank?: number
   likedByUserIds?: string[]
   likedByUserCount?: number
+  resolutionReports?: string[]
```

Comment:
This is a list of usernames? Maybe the name should be more explicit something like flaggedByUsernames